test/config/fetchmail.cf

```diff
@@ -9,3 +9,5 @@ poll pop3-2.example.com. with proto POP3
 	password 'secret'
 	is 'user3@domain.tld'
 	here options keep ssl
+
+# __DAEMON_INTERVAL__ 120
```

Comment:
> This is a 👎 from me.
> 
>     * Your issue (feature request) and PR have not clearly justified the need for the feature.
> 
>     * The current approach to implement it is not suitable for maintainers and may introduce a regression by reverting the comment stripping.
> 
> 
> I'm also a little reluctant towards adding niche features, especially when the value is unclear. I would prefer that we have more users wanting the feature or enabling you to more easily maintain the changes on your end, similar to `user-patches.sh`.
> 
> I am open towards a different approach that doesn't use a comment to configure it. Otherwise, I think we could extract out `_setup_fetchmail_parallel` to a separate file that is sourced, and you could override that, since presently you're the only user that seems to want this feature?

First, 👎 emoji looks too friendly, thanks.
As I'm trying to be a new contributor, and not familar with what you've done, thanks for pointing out problems in this pull request.
If there is a disaggressive way to achieve my goal, please let me know, maybe I can implement it the right way.